web/client/components/search/SearchBarUtils.js

```diff
@@ -6,6 +6,22 @@
  * LICENSE file in the root directory of this source tree.
 */
 
+const totalFeatures = (searchOptions, maxResults) => {
```

Comment:
can you add also jsdoc here?

```suggestion
const getTotalMaxResults = (searchOptions, maxResults) => {
```